ios/MarkdownCommitHook.mm

```diff
@@ -141,14 +141,12 @@
                     stateData.attributedStringBox);
 
             // Handles the first render, where the text stored in props is
-            // different than the one stored in state The one in state is empty,
-            // while the one in props is passed from JS If we don't update the
-            // state here, we'll end up with a one-default-line-sized text
-            // input. A better condition to do that can be probably chosen, but
-            // this seems to work
+            // different than the one stored in state. The one in state is empty,
+            // while the one in props is passed from JS. If we don't update the
+            // state here, we'll end up with a one-default-line-sized text input
             auto plainString =
```

Comment:
There's no need to call `std::string([[nsAttributedString string] UTF8String]);` if `textInputState.getRevision() != State::initialRevisionValue`, can we add another `if` outside to check if it's the first render?

Reply:
true, let me update …

Reply:
like this?

Reply:
yes, exactly, thanks!